account-service/src/main/java/com/service/accountservice/Controller/demo.java

```diff
@@ -1,17 +1,19 @@
 package com.service.accountservice.Controller;
 
+import org.springframework.http.HttpHeaders;
 import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+
 @RestController
 @RequestMapping("/api/v1/demo-controller")
 public class demo {
-
+    @CrossOrigin(origins = "http://localhost:3000")
     @GetMapping
     public ResponseEntity<String> sayHello() {
-        return ResponseEntity.ok("Hello from secured endpoint");
+        return ResponseEntity.ok("access granted");
```

Comment:
Just for future reference , this won’t be the actual api that will first be called when logging in, and entering the application. It probably would be the api that returns data from visualisation. But works for now